core/lib/src/fairing/fairings.rs

```diff
@@ -103,15 +103,16 @@ impl Fairings {
 
             let num = names.len();
             let joined = names.join(", ");
-            info_!("{} {}: {}", Paint::default(num).bold(), prefix, Paint::default(joined).bold());
+            info!("{} {}: {}", Paint::default(num).bold(), prefix, Paint::default(&joined).bold());
```

Comment:
It appears that `info!` expands the expression twice (once to emit a `log` event; once to emit a `tracing` event) - the call to `Paint::default()` moves the value, so it can only be done twice via a reference.